i18next-parser.config.js

```diff
@@ -8,7 +8,9 @@ module.exports = {
   defaultNamespace: "translation",
   // Default namespace used in your i18next config
 
-  defaultValue: "",
+  defaultValue(locale, namespace, key) {
```

Comment:
Bug since last bump of this dependency was leaving new strings with empty values